libsrc/meshing/meshtype.hpp

```diff
@@ -377,19 +377,19 @@ namespace netgen
 
   public:
     ///
-    Element2d () = default;
-    Element2d (const Element2d &) = default;
-    Element2d (Element2d &&) = default;
-    Element2d & operator= (const Element2d &) = default;
-    Element2d & operator= (Element2d &&) = default;
+    DLL_HEADER Element2d () = default;
```

Comment:
When a whole class is exported, inline member functions are not inlined anymore. Therefore we prefer exporting members explicitly. See [here](https://stackoverflow.com/questions/15676873/inline-class-functions-and-shared-library-dll-build#answer-15713486) for example.